views/settings/partials/rocket-insights/rocket-insights-column.php

```diff
@@ -0,0 +1,183 @@
+<?php
+/**
+ * Rocket Insights column content for post listing pages.
+ *
+ * @since 3.20.1
+ *
+ * @var array $data {
+ *     Template data.
+ *
+ *     @type string      $wpr_rocket_insights_url        The URL of the post.
+ *     @type object|null $wpr_rocket_row        Database row object for the URL (null if not tracked).
+ *     @type bool        $wpr_has_credit Whether the user has credit available.
+ * }
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+$wpr_rocket_insights_url = $data['url'] ?? ''; // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedVariableFound
+$wpr_rocket_row          = $data['row'] ?? null; // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedVariableFound
+$wpr_has_credit          = $data['has_credit'] ?? false; // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedVariableFound
```

Comment:
This should be done at the render controller level, that way there is no processing in the view